modules/installation-guide/partials/proc_configuring-labels-for-routes-ingresses.adoc

```diff
@@ -0,0 +1,40 @@
+
+[id="configuring-labels-for-routes_ingresses_{context}"]
+= Configuring labels for routes or ingresses
+
+This procedure describes how to configure labels for routes or ingresses to organize and categorize (scope and select) objects.
```

Comment:
Rebase on master to have the new attribute `{platforms-ingress}` defined in the branch and:

```suggestion
This procedure describes how to configure labels for {platforms-ingress} to organize and categorize (scope and select) objects.
```